diffkemp/cli.py

```diff
@@ -119,6 +119,13 @@ def make_argument_parser():
                             help="compare only selected function")
     compare_ap.add_argument("--custom-patterns", "-p",
                             help="custom pattern file or configuration")
+    compare_ap.add_argument("--c-pattern-append", help="option to append to \
+                            clang when compiling custom C pattern",
+                            action="append")
+    compare_ap.add_argument("--c-pattern-kernel-path",
```

Comment:
I'd omit the "c" part here and use the same name as for the `compile-pattern` command (see below), just with a `--pattern-` prefix.